client/src/utils/nullErrorMessage.js

```diff
@@ -0,0 +1,6 @@
+export const nullMessage = (errorMessage) => {
```

Comment:
will it be better if we change the name of this function to
`clearMessageTimeout` ?
does this name say more about what is going inside the function?

Reply:
Yes I changed